lib/auth/bot.go

```diff
@@ -258,6 +327,30 @@ func (a *Server) updateBotInstance(
 			return trace.Wrap(err)
 		}
 
+		// TODO: consider making this the only path where bot instances are
```

Comment:
maybe add username in TODO so we know who wrote it :)

Reply:
Good call, added!